templates/core/terraform/api-webapp/api-webapp.tf

```diff
@@ -38,10 +38,16 @@ resource "azurerm_app_service" "management_api" {
     "STATE_STORE_KEY"                       = var.state_store_key
     "SERVICE_BUS_FULLY_QUALIFIED_NAMESPACE" = "sb-${var.tre_id}.servicebus.windows.net"
     "SERVICE_BUS_RESOURCE_REQUEST_QUEUE"    = var.service_bus_resource_request_queue
-    TRE_ID                                 = "${var.tre_id}"
+    "MANAGED_IDENTITY_CLIENT_ID"            = "${var.managed_identity.client_id}" 
+    TRE_ID                                  = "${var.tre_id}"
```

Comment:
No need for the dollar sign and curly brackets here.

Reply:
Extra stuff removed